### PR TITLE
PIP: #240 Code optimization.

### PR DESCRIPTION
**Pull request by zjsunzone**
PIP: #240

### CODE REVIEW

File: consensus/cbft/network/peer_test.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func Test_PeerSet_Unregister(t *testing.T) {
 	err = ps.Unregister(p3.id)
 	assert.Equal(t, err.Error(), errNotRegistered.Error())
 
-	_, err = ps.Get(p3.id)
+	_, err = ps.get(p3.id)
```
Comment (JianhaiboWX) [inline consensus/cbft/network/peer_test.go]: Could "id" in "p3.id" be refactored to "ID"?

File: consensus/cbft/utils/util.go [hunk 2]
```diff
@@ -69,22 +69,23 @@ type KeyValuePair struct {
...
-func (h *KeyValuePairList) Push(x interface{}) {
-	*h = append(*h, x.(KeyValuePair))
+func (p *KeyValuePairList) Push(x interface{}) {
+	*p = append(*p, x.(KeyValuePair))
 }
 
-func (h *KeyValuePairList) Pop() interface{} {
-	old := *h
+func (p *KeyValuePairList) Pop() interface{} {
+	old := *p
```
Comment (JianhaiboWX) [inline consensus/cbft/utils/util.go]: Is it better to factor or rename "p" to "kvpl" as to represent KeyValuePairList ?

File: consensus/cbft/network/peer_test.go (hunk 1 above)
Comment (zjsunzone) [reply consensus/cbft/network/peer_test.go]: The scope of the id is private.

File: consensus/cbft/utils/util.go (hunk 2 above)
Comment (zjsunzone) [reply consensus/cbft/utils/util.go]: Thanks for the suggestion, I will optimize.

**Pull request merged**